Jenkinsfile

```diff
@@ -54,10 +47,20 @@ node('rhel8'){
         }
     }
 
+    env.publishPreReleaseFlag = ""
+    if(publishPreRelease.equals('true')){
```

Comment:
In a few other cases where you are checking if a variable is equal to `'true'`, you do `"true".equals(variable)`. Maybe it's better to do that here as well?

Reply:
Right. May as well reverse it. We don't in vscode-xml, but I think I just saw it used somewhere in the build files and decided to stick to that.